src/Refitter/Settings.cs

```diff
@@ -210,4 +210,9 @@ The NSwag IOperationNameGenerator implementation to use.
     [CommandOption("--use-dynamic-querystring-parameters")]
     [DefaultValue(false)]
     public bool UseDynamicQuerystringParameters { get; set; }
-}
\ No newline at end of file
+
+    [Description("Set to <c>true</c> to use System.Text.Json polymorphic serialization.")]
+    [CommandOption("--use-system-text-json-polymorphic-serialization")]
```

Comment:
@ryanheath Can we rename the command option to be `--use-polymorphic-serialization` ?